packages/MarkdownEditor-Core.package/MarkdownInlineParser.class/instance/tokenize.st

```diff
@@ -0,0 +1,10 @@
+parsing
+tokenize
+
+	| markdownStream |
+	self tokens: OrderedCollection new.
+	markdownStream := MarkdownReadStream on: self text asString.
+	[markdownStream atEnd] whileFalse: [| token |
+		token := self nextTokenIn: markdownStream.
+		self tokens addLast: token.
```

Comment:
You could also use self tokens add: token, because add: uses addLast:, but if you want to explicitly say where it gets added, you can keep it

Reply:
I think it's a good idea to explicitly say that we are adding it last because if it wouldn't be added last then the whole thing could break